owlbot.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
```

Comment:
Are we still using synthtool under the hood?

Reply:
go/owlbot-howto 
This doc will give you all the information regarding owlbot and how it works. The related docs at the end of this doc explains how changes are propagated to a repo through owlbot.
Always feel free to reach out to me if you have more questions.